### PR TITLE
feat(core/respec-global): add toHTML() method

### PR DESCRIPTION
**Pull request by sidvishnoi**
Part of https://github.com/w3c/respec/issues/3089

### CODE REVIEW

File: src/core/respec-global.js [hunk 1]
```diff
@@ -40,6 +41,10 @@ class ReSpec {
   get ready() {
     return this._respecDonePromise;
   }
+
+  async toHTML() {
```
Comment (sidvishnoi) [inline src/core/respec-global.js:45]: Async in case we need it in future

**Review by marcoscaceres — APPROVED** [review]

**Comment by sidvishnoi** [comment]
Should add a test too 😅

File: tests/spec/SpecHelper.js [hunk 2]
```diff
@@ -49,7 +49,7 @@ export function makeRSDoc(opts, src, style = "") {
     } else {
       const doc = document.implementation.createHTMLDocument();
       decorateDocument(doc, opts);
-      ifr.srcdoc = doc.documentElement.outerHTML;
+      ifr.srcdoc = `<!DOCTYPE html>${doc.documentElement.outerHTML}`;
```
Comment (marcoscaceres) [inline tests/spec/SpecHelper.js:52]: Naturally! :D

**Review by marcoscaceres — APPROVED** [review]
Good to have at least a minimal test.

**Comment by sidvishnoi** [comment]
Lets keep this feature for next release. Keep refactors and minor fixes for this release.

**Review by marcoscaceres — APPROVED** [review]

File: src/core/respec-global.js (hunk 1 above)
Comment (marcoscaceres) [reply src/core/respec-global.js:45]: We can also add it in the future...

**Comment by marcoscaceres** [comment]
> Lets keep this feature for next release. Keep refactors and minor fixes for this release.

Ok, but don't let them sit for too long (unless this will have consequences downstream).

**Pull request merged**